src/mm-test-form-view/mm-test-form-view.html

```diff
@@ -0,0 +1,151 @@
+<!--
```

Comment:
Should mm-test-form-view be included in the PR and/or is it properly excluded from distribution builds?